config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml

```diff
@@ -1,12 +1,12 @@
 presubmits:
-  kubernetes-csi/csi-driver-windows-poc:
+  kubernetes-csi/csi-driver-host-path:
   - name: pull-kubernetes-csi-csi-driver-windows-poc
     always_run: false
     decorate: true
     extra_refs:
-    - org: jingxu97
+    - org: kubernetes
       repo: csi-driver-host-path
-      base_ref: windows-poc
+      base_ref: master
       path_alias: kubernetes-csi/csi-driver-host-path
```

Comment:
you can drop this extra_refs block, presubmit will by default clone your PR snapshot

Reply:
removed. PTAL, thanks!